repository/snowflake.go

```diff
@@ -229,20 +220,46 @@ func (s *Snowflake) GetTrackingData(
 	return result, nil
 }
 
-func buildGetDataQuery(table, key string, fields []string, offset, limit int) string {
-	sb := sqlbuilder.NewSelectBuilder()
+// IsTableExist check if table exist.
+func (s *Snowflake) IsTableExist(ctx context.Context, table string) (bool, error) {
+	rows, err := s.conn.QueryContext(ctx, fmt.Sprintf(queryIsTableExist, strings.ToUpper(table)))
+	if err != nil {
+		return false, err
+	}
+	defer rows.Close()
 
-	if len(fields) == 0 {
-		sb.Select("*")
-	} else {
-		sb.Select(fields...)
+	return rows.Next(), nil
+}
+
+// IsStreamExist check if stream exist.
+func (s *Snowflake) IsStreamExist(ctx context.Context, stream string) (bool, error) {
```

Comment:
I'd suggest `StreamExists` or something similar (IsStreamExist isn't quite correct grammatically).

Reply:
changed

Reply:
we don't need to have this method, it was removed